.changeset/chilly-lizards-dress.md

```diff
@@ -0,0 +1,7 @@
+---
+'@twilio-paste/website': minor
```

Comment:
Suggestion (blocking): no need to include the website here, it isnt versioned. But you do need to add core  as a patch change because chat composer is in core.